bcda/cli.go

```diff
@@ -157,3 +148,61 @@ func getCCLFFileMetadata(filePath string) (cclfFileMetadata, error) {
 
 	return metadata, nil
 }
+
+func importCCLFDirectory(filePath string) (success, failure int, err error) {
+	var cclf0, cclf8, cclf9 []cclfFileMetadata
+
+	err = filepath.Walk(filePath, sortCCLFFiles(&cclf0, &cclf8, &cclf9))
+	if err != nil {
+		return 0, 0, err
+	}
+	for _, file := range cclf8 {
+		//todo match cclf0 and validate
+		err = importCCLF8(file)
+		if err != nil {
+			failure += 1
+			continue
+		}
+		log.Info(fmt.Sprintf("Successfully imported CCLF8 file: %v", file))
+		success += 1
+	}
+	for _, file := range cclf9 {
+		//todo match cclf0 and validate
+		err = importCCLF9(file)
+		if err != nil {
+			failure += 1
+			continue
+		}
+		log.Info(fmt.Sprintf("Successfully imported CCLF9 file: %v", file))
+		success += 1
+	}
+
+	return success, failure, nil
+}
+
+func sortCCLFFiles(cclf0, cclf8, cclf9 *[]cclfFileMetadata) filepath.WalkFunc {
+	return func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		// Directories are not CCLF files
+		if info.IsDir() {
+			return nil
```

Comment:
Should we log if we encounter this?  Maybe we received the files in a format we weren't expecting (i.e., wrapped in a dir).

Reply:
Also, do we have a test case for this?

Reply:
I made folders with CCLF like file names.  It looks like git might have just ignored them.  I will move some things around to get it to be committed.